mmv1/products/auditmanager/FrameworkAudit.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: FrameworkAudit
+base_url: "{{parent}}/frameworkAudits"
+description: Represents a framework audit.
+create_verb: :POST
```

Comment:
It looks like you might be using docs that pre-date our move off of Ruby. You no longer need the `:` here or in other places, or the `ruby/object` declarations, etc. Please refer to https://googlecloudplatform.github.io/magic-modules/reference/ruby-go-changes/ for more details.

Fixing these will probably fix most of your bugs.